graylog2-web-interface/src/views/logic/queries/TimeRangeToString.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import { AbsoluteTimeRange, KeywordTimeRange, RelativeTimeRange, TimeRange } from 'views/logic/queries/Query';
+import { isTypeRelative } from 'views/typeGuards/timeRange';
+import DateTime from 'logic/datetimes/DateTime';
+
+export const readableRange = (timerange: TimeRange, fieldName: 'range' | 'from' | 'to', placeholder = 'All Time') => {
+  return !timerange[fieldName] ? placeholder : DateTime.now()
+    .subtract(timerange[fieldName] * 1000)
+    .fromNow();
+};
+
+const relativeTimeRangeToString = (timerange: RelativeTimeRange): string => {
+  if (isTypeRelative(timerange)) {
+    return readableRange(timerange, 'range');
+  }
+
+  return `${readableRange(timerange, 'from')} - ${readableRange(timerange, 'to')}`;
+};
```

Comment:
I saw that a widget time range like `5 minutes ago - Now` results in `All Time`.
We can fix it with:


```suggestion
const relativeTimeRangeToString = (timerange: RelativeTimeRange): string => {
  if (isTypeRelativeWithStartOnly(timerange)) {
    return `${readableRange(timerange, 'range')} - Now`;
  }

  return `${readableRange(timerange, 'from')} - ${readableRange(timerange, 'to', 'Now')}`;
};

```

With the previous time range picker changes the widget time range no longer contains a `range` attribute, but I see a benefit in taking care of this case, as well.